static/sass/_snapcraft_release.scss

```diff
@@ -17,30 +17,67 @@
     margin-left: auto;
   }
 
-  .p-releases-confirm__rollout {
+  .p-releases-confirm__details-toggle {
+    cursor: pointer;
     display: flex;
     flex-grow: 1;
     justify-content: flex-end;
     margin-right: 1rem;
 
-    label {
-      margin: 0;
-      padding: 0;
+    .p-icon--contextual-menu {
+      transform: rotate(0deg);
     }
 
-    &-percentage {
-      margin-bottom: 0;
-      margin-right: .5rem;
-      min-width: 4rem;
-      width: 4rem;
+    &.is-open {
+      .p-icon--contextual-menu {
+        transform: rotate(180deg);
+      }
     }
   }
 
-  .p-releasses-confirm__actions {
+  .p-releases-confirm__actions {
     display: flex;
     justify-content: space-between;
   }
 
+  .p-releases-confirm__details {
+    border-bottom: 1px solid $color-mid-light;
+    margin: .5rem 0 .5rem;
+    padding-bottom: .5rem;
+  }
+
+  .p-releases-confirm-details__row {
+    align-items: center;
+    display: grid;
+    font-size: .875rem;
+    grid-column-gap: 1rem;
+    grid-template-columns: 4rem 15rem 2rem auto 16rem;
+    padding: 1rem;
+
+    &.is-closing {
+      grid-template-columns: auto;
+    }
+
+    &:nth-child(odd) {
+      background: $color-light;
+    }
+
+    &-join {
```

Comment:
If it contains sub-elements maybe it should be a top level pattern of its own, rather than `__row` in details?
This gets very nested.

But I wont help with naming all those things ;) So kinda understand why it happened...